metadatastore/commands.py

```diff
@@ -169,11 +326,9 @@ def insert_run_start(time, scan_id, beamline_id, beamline_config, uid=None,
     if project is None:
         project = ''
 
-    beamline_config = _get_mongo_document(beamline_config, BeamlineConfig)
     run_start = RunStart(time=time, scan_id=scan_id,
-                         time_as_datetime=_todatetime(time), uid=uid,
```

Comment:
Because this is something that should be taken care of on the way out (see the companion dataportal PR) and is already gone from the ODM templates.

This also makes @arkilic 's life easier by keeping the datatypes stored in the db simple.